integrationTests/chainSimulator/mempool/mempool_test.go

```diff
@@ -1,18 +1,26 @@
 package mempool
 
 import (
+	"fmt"
+	"math"
 	"math/big"
 	"testing"
 	"time"
 
+	"github.com/multiversx/mx-chain-core-go/data/block"
 	"github.com/multiversx/mx-chain-core-go/data/transaction"
+	"github.com/multiversx/mx-chain-go/common/holders"
+	"github.com/multiversx/mx-chain-go/testscommon/txcachemocks"
+	"github.com/multiversx/mx-chain-go/txcache"
 	"github.com/stretchr/testify/require"
 
 	"github.com/multiversx/mx-chain-go/config"
 	"github.com/multiversx/mx-chain-go/node/chainSimulator/configs"
 	"github.com/multiversx/mx-chain-go/storage"
 )
 
+var selectionLoopMaximumDuration = 500 // loosen the time constraints for selection integration tests
```

Comment:
Maybe we should have brought the changes from rc/supernova, towards the feat branch, then into this branch, instead?

Works this way, anyway - some conflicts will appear at some point, though.